stories/topics/proc-saas-set-up.adoc

```diff
@@ -0,0 +1,24 @@
+[id="proc-saas-set-up"]
+
+= Setting up {SaaSonAWS}
+
+Red{nbsp}Hat {SaaSonAWS} users will have access to {PlatformNameShort} without the need to manage the infrastructure. {SaaSonAWS} will deploy and configure a ROSA cluster for you. {SaaSonAWS} will deploy on top of your ROSA cluster, which enables you to run it on {OCP}.
```

Comment:
May be my lack of knowledge of the product but should ROSA be defined on first use?